packages/webview_flutter/example/integration_test/webview_flutter_test.dart

```diff
@@ -1146,8 +1146,48 @@ void main() {
       );
 
       expect(errorCompleter.future, doesNotComplete);
+      await Future.delayed(Duration(seconds: 5));
```

Comment:
I missed this when looking at the PR before; why is there a 5-second sleep at the end of these tests? At the very least this needs a clear comment, but is there really not some reliable thing we can wait for to do whatever this is intended to do, rather than relying on timing?

Reply:
I think I believed there was a chance for an error to occur after the page finished loading, but I just tested it and that doesn't look like the case for the url provided. I changed it to wait for `onPageFinished`.